tests/testdata/data-sources.md

```diff
@@ -0,0 +1,22 @@
+# Data Sources
+
+| Data Source         | Dataset name                      | Format    | License                            |
+|---------------------|-----------------------------------|-----------|------------------------------------|
+| [sf]                | nc                                | gpkg      | COPYRIGHT HOLDER: Edzer Pebesma    |
+| [sf]                | overpass                          | osm       | ODbL                               |
+| [Geosoftware2]      | clc_1000_PT                       | gml       | MIT                                |
+| [Geosoftware2]      | wf_100m_klas                      | geotif    | MIT                                |
+| [Geosoftware2]      | cities_NL                         | csv       | MIT                                |
+| [Geosoftware2]      | muenster_ring_zeit                | geojson   | MIT                                |
+| [Geosoftware2]      | schutzhuetten_aachen              | json      | MIT                                |
+| [Geosoftware2]      | ECMWF_ERA-40_subset1              | netcdf    | MIT                                |
+| [Geosoftware2]      | aasee                             | kml       | MIT                                |
+| [Geosoftware2]      | Abgrabungen_Kreis_Kleve_Shape     | shp       | MIT                                |
+| [Geosoftware2]      | wf_100m_klas                      | geotif    | MIT                                |
```

Comment:
It is very unlikely all of these files have the `MIT` license, as we discussed before. Please double check _with the original sources_, not with the GitHub repository where you got the files, and rather put "unknown" in there until we know.

Reply:
I updated the data source but some data-sets have no license.